package.json

```diff
@@ -1,5 +1,4 @@
 {
-  "name": "business-edit-ui",
```

Comment:
Wait, what?

Reply:
Wow, good catch, I didn't remove that line consciously.

Reply:
Done